gammapy/makers/tests/test_spectrum.py

```diff
@@ -83,8 +83,8 @@ def test_spectrum_dataset_maker_hess_dl3(spectrum_dataset_crab, observations_hes
     assert_allclose(datasets[0].exposure.meta["livetime"].value, 1581.736758)
     assert_allclose(datasets[1].exposure.meta["livetime"].value, 1572.686724)
 
-    assert_allclose(datasets[0].npred_background().data.sum(), 7.74732, rtol=1e-5)
-    assert_allclose(datasets[1].npred_background().data.sum(), 6.118879, rtol=1e-5)
+    assert_allclose(datasets[0].npred_background().data.sum(), 7.747881, rtol=1e-5)
+    assert_allclose(datasets[1].npred_background().data.sum(), 5.731624, rtol=1e-5)
```

Comment:
Here the change on the background counts seems quite large (~8%). Any idea why?

Reply:
Yes, in the previous implementation the background evaluation was forced to fov-lat=0, fov-lon = offset, assuming the background was radially symmetric. Now the correct position in fov coordinates is used.